#!/usr/bin/python3
def raise_exception():
    '2' + 2
